spec/services/timebox_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe Timebox do
+  subject { described_class.new(seconds) }
+
+  let(:result) do
+    subject.run(-> { :fellback }) do
+      sleep 0.2
+      :finished
+    end
+  end
+
+  context '#run' do
+    context 'big timebox' do
+      let(:seconds) { 10 }
```

Comment:
We don't actually wait 10 seconds - just the 0.2 seconds in the main block (line 8) - the 10 seconds is the param we put in when we instantiate Timebox to ensure we'll never trigger a fallback.